chapter04/Task_4_2.py

```diff
@@ -0,0 +1,8 @@
+# The program outputs the text in reverse
+
+text = input('Enter text\n')
+
+while text:
```

Comment:
Not sure if you had this in the book, but print(text[::-1]) should work.
The main advantages of this approach are that it's more readable (as it's less code) and faster (because less python operations are executed and instead C-based string function is called).
Catch up if you don't know why this works.